### PR TITLE
fixed tooltip not shown when button is disabled

### LINKED ISSUE #698: Tooltip not shown when button is disabled
neetoui

cc @edwinbbu

### PR DESCRIPTION
**Pull request by abhirampai**
fixes #698 

@edwinbbu  _a

### CODE REVIEW

File: lib/components/Button.js [hunk 1]
```diff
@@ -68,60 +68,65 @@ const Button = React.forwardRef((props, ref) => {
...
-        disabled={disabled}
-        {...elementSpecificProps}
-        {...otherProps}
-      >
-        {label && <span>{label}</span>}
+    <Tooltip
+      {...tooltipProps}
+      disabled={!tooltipProps || tooltipProps.disabled}
+    >
+      <span className="inline-block">
```
Comment (edwinbbu) [inline lib/components/Button.js:75]: @abhirampai Why is this change required.

Comment (abhirampai) [reply lib/components/Button.js:75]: @edwinbbu  or else the tooltip won't be disabled even if tooltipProps has disabled true.

Comment (abhirampai) [reply lib/components/Button.js:75]: tooltip will be disabled only if tooltipProps is not passed. if this rule is not added.

Comment (edwinbbu) [reply lib/components/Button.js:75]: @ajmaln _a Can you take a look just to verify.

File: lib/components/Button.js [hunk 2]
```diff
@@ -68,60 +68,65 @@ const Button = React.forwardRef((props, ref) => {
...
-        })}
-        disabled={disabled}
-        {...elementSpecificProps}
-        {...otherProps}
-      >
-        {label && <span>{label}</span>}
+    <Tooltip
+      {...tooltipProps}
+      disabled={!tooltipProps || tooltipProps.disabled}
+    >
```
Comment (ajmaln) [inline lib/components/Button.js:74]:
```suggestion
    <Tooltip
      disabled={!tooltipProps}
      {...tooltipProps}
    >
```
@abhirampai _a can you try this instead?

Comment (abhirampai) [reply lib/components/Button.js:74]: @ajmaln yes this works.

**Comment by amaldinesh7** [comment]
LGTM!

**Pull request merged**